src/gtimelog/timelog.py

```diff
@@ -656,7 +659,7 @@ def _plain_report(self, output, email, who, subject, period_name):
         categories = {}
         if work:
             work = [(entry, duration) for start, entry, duration in work]
-            work.sort()
+            work.sort(reverse=True, key=by_duration)
```

Comment:
This is nice, but by_duration is not useful in other contexts, so it ought to be defined here, or, even better, you can do

```python
work = sorted(((duration, entry) for start, entry, duration in work), reverse=True)
```

and swap `entry, duration` in the for loop below.